modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -40,28 +62,67 @@ metadata:
   name: elasticsearch
   namespace: openshift-logging
 spec:
-  host:
+  host: 
   to:
     kind: Service
     name: elasticsearch
   tls:
-    termination: reencrypt <1>
-    destinationCACertificate: |-   <2>
-      -----BEGIN CERTIFICATE-----
-      [...]
-      -----END CERTIFICATE-----
+    termination: reencrypt
+    destinationCACertificate: <1>
 ----
-<1> Set the termination field to reencrypt.
-<2> Add the contents of the *_admin-ca_* file.
+<1> Add the Elasticsearch CA ceritifcate or use the command in the next step. You do not need to set the `spec.tls.key`, `spec.tls.certificate` and `spec.tls.caCertificate` parameters
+required by some reencrypt routes.
 
-. Use the following command to create the service:
+.. Run the following commnd to add the Elasticsearch CA certificate to the route YAML you created:
 +
 ----
-$ oc create -f my_es_route.yaml
+$cat ./admin-ca | sed -e "s/^/      /" >> <file-name>.yaml
+----  
+
+.. Run the following command to create the route:
++
+----
+$oc create -f <file-name>.yaml
+
+route.route.openshift.io/elasticsearch created
 ----
++
+For an example reencrypt route object, see Re-encryption Termination.
++
+//This line ^^ will be linked when the topic is available.
+
+. Check that the Elasticsearch service is exposed:
 
-. Check that the ElasticSearch service is exposed:
+.. Get the token of this ServiceAccount to be used in the request:
 +
 ----
-curl --silent --insecure -H "Authorization: Bearer $( oc whoami -t )" "https://$( oc get route elasticsearch -o jsonpath='{.spec.host}' ):443/.operations.*/_search" | jq
+$ token=$(oc whoami -t)
 ----
+
```

Comment:
Please add this line.
`$ elasticsearch_fqdn=$(oc get route elasticsearch -o jsonpath='{.spec.host}')`

Reply:
What does this command do? Set the route as a FQDN?

Reply:
It retrieves the elasticsearch fqdn from the newly created elasticsearch route.  I saw you put my test fqdn in the curl command line below and I wanted to make it more generic for the readers.